docs/installation/compatibility.md

```diff
@@ -0,0 +1,53 @@
+
+This page provides information about Liqo's compatibility with different Kubernetes providers.
+
+```{admonition} Note
+While the following list includes providers that we have specifically tested, Liqo should work with any Kubernetes-compliant distribution, although some provider-specific configurations might be required.
+```
+
+## Legend
+
+- ✅ Fully supported - All features work as expected
+- 🟢 Mostly supported - Core features work well, with only minor limitations in specific scenarios
+- 🔵 Partial support - Some Liqo features are functional, but others may require specific configurations or have major limitations in certain use cases
+
+## Tested Provider Compatibility Table
+
+| Provider | Status | Known Issues |
+|----------|--------|--------------|
+| Kubeadm (Calico) | ✅ | No known issues |
+| Kubeadm (Cilium) | ✅ | No known issues |
+| Kubeadm (Cilium with kube-proxy replacement) | 🟢 | `NodePortExposition` and `LoadBalancerExposition` |
+| K3s | 🟢 | `RemoteExec` |
+| K0s | ✅ | No known issues |
+| AKS (Azure CNI Overlay) | 🟢 | `CrossClusterAPIServerInteraction` and `ExternalIPRemapping` |
+| AKS (Azure CNI (Legacy)) | 🟢 | `CrossClusterAPIServerInteraction` and `NodePortExposition` |
+| AKS (Kubenet) | 🟢 | `CrossClusterAPIServerInteraction` and `ExternalIPRemapping` |
+| EKS | 🟢 | `CrossClusterAPIServerInteraction` and `ExternalIPRemapping` |
+| GKE (Dataplane v1) | ✅ | No known issues |
+| GKE (Dataplane v2) | 🟢 | `NodePortExposition` and `LoadBalancerExposition` |
+| Aruba Cloud KaaS | ✅ | No known issues |
+| OpenShift | 🔵 | **Work in progress**: all Liqo functionalities except for the networking module work as expected. The team is actively working on adding full networking support. You can still use it by [disabling the Network Module](AdvancedUseOnlyOffloadingDisableModule). |
+| *Your K8s Distribution* | 🟢 | Liqo is designed to work with most Kubernetes-compliant distributions. Your provider is likely supported, but may require specific configurations. |
+
+### Help Us Improve
+
+Have you tested Liqo with a provider not listed here?
+We'd love to hear about your experience!
+Join our [Slack community](https://liqo-io.slack.com/join/shared_invite/zt-h20212gg-g24YvN6MKiD9bacFeqZttQ) and share your test results.
+Your feedback helps us improve Liqo's compatibility across different environments.
+
+## Issues Reference
+
+The following issues are known to affect the compatibility of Liqo with different Kubernetes providers:
+
+- `CrossClusterAPIServerInteraction`: The ability of offloaded pods to properly interact with the Kubernetes API server of the home cluster. This ensures that applications running in remote clusters can still access and manipulate Kubernetes resources (such as ConfigMaps, Secrets, or other custom resources) in their original cluster. Limitations in this area may impact applications that rely on the Kubernetes API for normal operation. See [here](../advanced/kubernetes-api.md) for more details.
```

Comment:
what about talking about "consumer" cluster or "remote cluster"? Home cluster is a definition that has been deprecated if I remember well.